app/models/embed/purl.rb

```diff
@@ -2,6 +2,7 @@ module Embed
   class PURL
     require 'dor/rights_auth'
     delegate :embargoed?, to: :rights
+    delegate :stanford_only_unrestricted?, to: :rights
```

Comment:
I think this could be in one delegate line.  Not sure if it reads better that way or this way, so totally up to you.